lib/routes/sketis/isabelle-dev/blogs/namespace.ts

```diff
@@ -0,0 +1,13 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'isabelle-dev',
+    url: 'https://isabelle-dev.sketis.net/phame/',
+    description: `
+:::tip
+Isabelle/Phabricator Blogs
+
+-   Isabelle Release: \`https://isabelle-dev.sketis.net/phame/blog/view/2/\`
+-   Isabelle News: \`https://isabelle-dev.sketis.net/phame/blog/view/1/\`
+:::`,
```

Comment:
These should descript the namespace
```suggestion
    name: 'Sketis | Website of Dr. Makarius Wenzel',
    url: 'sketis.net',
```

Reply:
I'm changing the namespace to `isabelle-dev`, which is more approperate for this situation.

Reply:
You can use RouteItem.description for route specific description.

> changing the namespace to `isabelle-dev`

In principle, the namespace should be **the same** as the secondary domain of the main website where you are making the RSS feed. For example, if you are making an RSS feed for https://github.com/DIYgod/RSSHub/issues, the secondary domain is `github`.

docs: https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace